tests/src/test/java/docs/javadsl/ConsumerExampleTest.java

```diff
@@ -532,9 +532,9 @@ public void onStop(Set<TopicPartition> currentTps, RestrictedConsumer consumer)
     assertDone(control.isShutdown());
     assertEquals(messageCount, resultOf(control.drainAndShutdown(executor)).size());
 
-    assertThat(revoked.get(), is(Collections.emptySet()));
     assertThat(assigned.get(), hasItem(tp));
     assertThat(stopped.get(), hasItem(tp));
+    assertThat(revoked.get(), hasItem(tp)); // revoke of partitions occurs after stop
```

Comment:
Does this mean, we could do without the `onStop` addition in the `PartitionAssignmentHandler`? I added it as there was no revoking when the consumer is stopping.

Reply:
Possibly.  I quickly scanned release notes from 2.2.0 to 2.4.0 and I'm not sure when this change occurred.

Reply:
https://github.com/akka/alpakka-kafka/issues/982